src/tlo/methods/healthsystem.py

```diff
@@ -2473,6 +2515,8 @@ def _reset_internal_stores(self) -> None:
         self._never_ran_appts_by_level = {_level: defaultdict(int) for _level in ('0', '1a', '1b', '2', '3', '4')}
 
         self._frac_time_used_overall = []  # Running record of the usage of the healthcare system
+        self._frac_time_used_by_officer_type_and_level = Counter()
```

Comment:
```suggestion
        self._sum_frac_time_used_by_officer_type_and_level = Counter()
```
I think prefixing with `sum` here makes it more explicit this is a sum of fractions (see also suggested change above).